main.py

```diff
@@ -59,25 +83,30 @@ def generate(lang_in, test_len=None, make_maps=True, map_img_type='svg'):
     # meta = clean_xls_headers(meta, 1)
     # faq = clean_xls_headers(faq, 1)
 
+    palika_codes = data.sht.index[:test_len] if test_len else data.sht.index
+
     if make_maps:
-        gen_maps(list(data.sht.index), map_img_type)
+        gen_maps(list(palika_codes), map_img_type)
 
     # process
     PATH = './output/%s_%s.pdf'
-    for v in data.sht.index.values[:test_len] if test_len else data:
-        if (not skip or str(v) not in skip): #and not os.path.isfile(PATH % (v, lang_in)):
+    for v in palika_codes:
+        if (not skip or str(v) not in skip):  # and not os.path.isfile(PATH % (v, lang_in)): # noqa
             print('Creating profile for %s' % v)
             cur_rep = Report(gc=v, data_sht=data.sht, meta_sht=meta.sht,
                              faq_sht=faq.sht, map_img_type=map_img_type)
             cur_rep.create_data()
 
             os.makedirs('./output/', exist_ok=True)
-            PdfDraft(PATH % (v, lang_in)) \
+            pdf_draft = PdfDraft(PATH % (v, lang_in))\
                 .draw(Page1(cur_rep.data, lang_in))
-                # .draw(Page2(cur_rep.data, lang_in))
+            if make_maps:
```

Comment:
going to take this out as we may we want to make the second page without having to re-generate the maps. a boolean for make_second would be better